src/component/DocumentItem.js

```diff
@@ -0,0 +1,126 @@
+import Button from "../common/Button.js";
+import Title from "../common/Title.js";
+import arrowIconSvg from "../svg/arrowIcon.js";
+import plusIcon from "../svg/plusIcon.js";
+import xIcon from "../svg/xIcon.js";
+import { push } from "../utils/router.js";
+import Storage from "../utils/storage.js";
+import DocumentList from "./DocumentList.js";
+import NoSubDocument from "./NoSubDocument.js";
+
+export default function DocumentItem({
+  $target,
+  initialState,
+  createDocument,
+  removeDocument,
+  depth,
+  changeBackgroundSelectedDocument,
+}) {
+  this.state = initialState;
+  const $documentItem = document.createElement("div");
+  const $documentItemInner = document.createElement("div");
+  $documentItemInner.style.paddingLeft = `${depth > 0 ? depth * 10 : 10}px`;
+  $target.appendChild($documentItem);
+  $documentItem.dataset.id = this.state.id;
+  $documentItem.classList.add("document-item");
+  $documentItemInner.classList.add("document-item-inner");
+  const storage = new Storage(window.localStorage);
+  const getChildDocuments = () => {
+    const childDocuments = $documentItem.querySelector(".document-children");
+    if (childDocuments) {
+      return childDocuments;
+    }
+    return undefined;
+  };
```

Comment:
요소를 찾을수없으면 undefined를 반환하지않나요?
```suggestion
  const getChildDocuments = () => $documentItem.querySelector(".document-children");
```

Reply:
찾아보니 `null`을 반환합니다!
하지만 `falsy`한값이므로 예외처리를 지워주고 중괄호도 지워주었습니다
없을때는 어떤값이 반환되는지 잘 살펴보도록 하겠습니다!